js/popup.js

```diff
@@ -0,0 +1,55 @@
+const yearEl = document.querySelector('.year');
+const headerEl = document.querySelector('.header');
+const clientsEl = document.querySelector('.clients');
+const btnOpenEl = document.querySelector('.icon-open');
+const btnCloseEl = document.querySelector('.icon-close');
+
+yearEl.textContent = new Date().getFullYear();
+
+btnOpenEl.addEventListener('click', () => {
+  headerEl.classList.add('nav-open');
+  document.body.style.overflow = 'hidden';
+});
+
+btnCloseEl.addEventListener('click', () => {
+  headerEl.classList.remove('nav-open');
+  document.body.style.overflow = 'scroll';
+});
+
+const clientsInfo = [
+  {
+    link: 'https://www.netflix.com',
+    imgUrl: ['../imgs/netflix.svg', 'Netflix logo'],
+  },
+  {
+    link: 'https://www.google.com',
+    imgUrl: ['../imgs/google.svg', 'Google logo'],
+  },
+  {
+    link: 'https://www.cocacola.com',
+    imgUrl: ['../imgs/cocacola.svg', 'CocaCola logo'],
+  },
+  {
+    link: 'https://www.microsoft.com',
+    imgUrl: ['../imgs/microsoft.svg', 'Microsoft logo'],
+  },
+  {
+    link: 'https://www.paypal.com',
+    imgUrl: ['../imgs/paypal.svg', 'PayPal logo'],
+  },
```

Comment:
- [ ] Same issue with images as discussed above. Please worker with the suggestion below.

  ```suggestion
    {
      link: 'https://www.netflix.com',
      imgUrl: ['./imgs/netflix.svg', 'Netflix logo'],
    },
    {
      link: 'https://www.google.com',
      imgUrl: ['./imgs/google.svg', 'Google logo'],
    },
    {
      link: 'https://www.cocacola.com',
      imgUrl: ['./imgs/cocacola.svg', 'CocaCola logo'],
    },
    {
      link: 'https://www.microsoft.com',
      imgUrl: ['./imgs/microsoft.svg', 'Microsoft logo'],
    },
    {
      link: 'https://www.paypal.com',
      imgUrl: ['./imgs/paypal.svg', 'PayPal logo'],
    },
  ```
  
  When the app is finally deployed, the server expects to work with the absolute path provided but it's not finding any image in the location we are providing so far.

Reply:
Noted. I will work on it.